src/metax_api/tests/api/rest/base/views/directories/read.py

```diff
@@ -354,9 +354,10 @@ def test_read_directory_for_catalog_record_recursively(self):
         response = self.client.get('/rest/directories/1/files?recursive&cr_identifier=%s&depth=*'
             % CatalogRecord.objects.get(pk=1).identifier)
         self.assertEqual(response.status_code, status.HTTP_200_OK)
-        self.assertEqual(len(response.data), 2)
+        cr_list = list(File.objects.filter(record__pk=1).values_list('id', flat=True))
```

Comment:
Is ``cr_list`` a list of files or a list of catalog records? Variable name implies list of catalog records?

Reply:
somehow with this ticket have been mixing all the time catalog_records and directories....